packages/fxa-shared/db/models/auth/session-token.ts

```diff
@@ -264,7 +266,8 @@ export class SessionToken extends BaseToken {
   static async findByUid(uid: string): Promise<SessionToken[]> {
     const { rows } = await this.callProcedure(
       Proc.Sessions,
-      uuidTransformer.to(uid)
+      uuidTransformer.to(uid),
+      GET_SESSIONS_QUERY_LIMIT,
```

Comment:
Just a nitpick. Since this is in fxa-shared it can potentially be used by more than the auth server, so perhaps it shouldn't make assumptions about how the code is used. Making this an optional parameter with a default value would make this limit clear to any calling code. It would also allow greater flexibility if for some reason a high device count was needed. (e.g. An administrative operation or a clean up script was needed.)